src/useGet.tsx

```diff
@@ -56,6 +56,13 @@ export interface UseGetProps<TData, TQueryParams> {
     | number;
 }
 
+function resolvePath<TQueryParams>(base: string, path: string, queryParams: TQueryParams) {
+  const escapedBase = base.endsWith("/") ? base : `${base}/`;
+  const escapedPath = path.startsWith("/") ? path.slice(1) : path;
```

Comment:
`escaped` confuses me here - makes me think about url encoding, how about `appendedBase` and `trimmedPath`?